src/Jackett.Common/Indexers/PolishTracker.cs

```diff
@@ -83,105 +83,87 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
         {
             var releases = new List<ReleaseInfo>();
 
-            var searchUrl = TorrentApiUrl;
-            var searchString = query.GetQueryString();
-            var queryCollection = new List<KeyValuePair<string, string>>
+            var qc = new List<KeyValuePair<string, string>> // NameValueCollection don't support cat[]=19&cat[]=6
```

Comment:
See #7915 for a fix for this. I was already working on it for my own refactoring.